framework/doc/content/source/postprocessors/SideAdvectiveFluxIntegral.md

```diff
@@ -0,0 +1,20 @@
+# SideAdvectiveFluxIntegral
+
+!syntax description /Postprocessors/SideAdvectiveFluxIntegral
+
+## Description
+
+The `SideAdvectiveFluxIntegral` postprocessor is used to compute the side integral of the advection flux component including x, y, z and normal. This kernel supports both finite element and finite volume variables. For finite element method, the variable needs to be a continuous finite element type(e.g.LAGRANGE, HIERARCHIC, BERNSTEIN, SZABAB, RATIONAL_BERNSTEIN, CLOUGH, HERMITE, etc.), so that the side value can be estimated in MOOSE. The advection variable can be either a variable[!param](/Postprocessors/SideAdvectiveFluxIntegral/advected_variable) or material property[!param](/Postprocessors/SideAdvectiveFluxIntegral/advected_mat_prop). For finite volume method, an advected quantity needs to be provided[!param](/Postprocessors/SideAdvectiveFluxIntegral/advected_quantity). The advection velocity is required with [!param](/Postprocessors/SideAdvectiveFluxIntegral/vel_x) for advection flux calculation. [!param](/Postprocessors/SideAdvectiveFluxIntegral/vel_y) and [!param](/Postprocessors/SideAdvectiveFluxIntegral/vel_z) are needed for 2D and 3D simulations.
```

Comment:
```suggestion
The `SideAdvectiveFluxIntegral` postprocessor is used to compute the side integral of the advection flux component including x, y, z and normal. This kernel supports both finite element and finite volume variables.

For the finite element method, the variable needs to be a continuous finite element type (e.g. LAGRANGE, HIERARCHIC, others), so that the side value can be estimated in MOOSE. The advection variable can be either a variable[!param](/Postprocessors/SideAdvectiveFluxIntegral/advected_variable) or material property[!param](/Postprocessors/SideAdvectiveFluxIntegral/advected_mat_prop).

For the finite volume method, an advected quantity needs to be provided using the [!param](/Postprocessors/SideAdvectiveFluxIntegral/advected_quantity) parameter. The advection velocity is required with [!param](/Postprocessors/SideAdvectiveFluxIntegral/vel_x) for advection flux calculation. [!param](/Postprocessors/SideAdvectiveFluxIntegral/vel_y) and [!param](/Postprocessors/SideAdvectiveFluxIntegral/vel_z) are needed for 2D and 3D simulations.
```